llvm/test/Transforms/LoopVectorize/AArch64/wide-trip-count.ll

```diff
@@ -0,0 +1,33 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
```

Comment:
Does it make sense to move this test into the existing Transforms/LoopVectorize/AArch64/clamped-trip-count.ll file?

Reply:
Hm, you make a good point, yeah I'll give that a go.

Reply:
Done